inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -99,7 +99,10 @@ def populate_arxiv_document(obj, eng):
         original_url=url,
         url=url,
     )
+
     obj.data = lb.record
+    if not obj.data.get('documents') and 'documents':
```

Comment:
same (and the check is wrong here btw, you forgot `'documents' in obj.data` I think)